### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by davidkyburz**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -6,21 +6,55 @@
...
+msgstr "Sprache"
+
+#: pdfarranger/config.py:221
+msgid "(Requires restart)"
+msgstr "(Erfordert Neustart)"
+
+#: pdfarranger/config.py:227
+msgid "Theme"
+msgstr "Thema"
+
```
Comment (dreua) [inline po/de.po:50]: I think this is about the color scheme, maybe we can have a look how this is translated and named in other applications.

File: po/de.po [hunk 2]
```diff
@@ -219,161 +253,166 @@ msgstr ""
...
+#: pdfarranger/pdfarranger.py:1055
 msgid "Export…"
 msgstr "Exportieren …"
 
-#: pdfarranger/pdfarranger.py:1051 data/menu.ui.h:1
+#: pdfarranger/pdfarranger.py:1098
+#, python-format
+msgid "A file named \"%s\" already exists. Do you want to replace it?"
+msgstr ""
+"Eine Datei mit dem Namen \"%s\" existiert bereits. Möchten Sie sie ersetzen?"
```
Comment (dreua) [inline po/de.po:349]: I think "Sie diese ersetzen" is more common and smoother.

**Review by dreua — APPROVED** [review]
Thanks for the good translations, I've commented on two strings which are okay but not perfect IMO. Would you like to improve those? I'm fine with merging it anyway.

Vielen Dank!

File: po/de.po (hunk 1 above)
Comment (davidkyburz) [reply po/de.po:50]: Yes, it's about the color scheme. I will change it to "Erscheinungsbild". This term is used in Firefox. Other ideas are "Stil" (Gnome) or "Modus" (Windows).

File: po/de.po (hunk 2 above)
Comment (davidkyburz) [reply po/de.po:349]: Thanks for pointing this out.

**Comment by dreua** [comment]
Perfekt, danke!

**Pull request merged**